common/src/main/java/party/lemons/biomemakeover/level/feature/PeatFeature.java

```diff
@@ -41,7 +47,7 @@ public boolean place(FeaturePlaceContext<NoneFeatureConfiguration> ctx) {
             {
                 int offsetX = xx - centerPos.getX();
                 int offsetZ = zz - centerPos.getZ();
-                if(offsetX * offsetX + offsetZ * offsetZ <= radius * radius)
+                if(offsetX * offsetX + offsetZ * offsetZ <= radius * (radius + 1))
```

Comment:
Integer equivalent to this https://www.redblobgames.com/grids/circle-drawing/#aesthetics